satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -377,6 +377,72 @@ read limitoffset (
     orderby desc bucket_usage.rollup_end_time
 )
 
+//--- used serials ---//
+
+model used_serial {
+	key    serial_number
+	unique serial_number storage_node_id
+
+	field serial_number   blob
+	field storage_node_id blob
+
+	field bucket_id  blob
+	field expires_at utimestamp
+}
+
+// --- bucket roll up tables --- //
+
+model bucket_bandwidth_rollup {
+	key    bucket_id
+	unique bucket_id interval_start interval_seconds action
+
+	field bucket_id        blob
+	field interval_start   utimestamp
+	field interval_seconds uint32
+	field action           byte
+
+	field inline    uint64
+	field allocated uint64
+	field settled   uint64
+}
+
+model bucket_storage_rollup (
+	key    bucket_id
+	unique bucket_id interval_start interval_seconds
+
+	field bucket_id        blob
+	field interval_start   utimestamp
+	field interval_seconds uint32
+
+	field inline uint64
+	field remote uint64
+)
+
+// --- storage node roll up tables --- //
+
+model storagenode_bandwidth_rollup {
+	key    storagenode_id
+	unique storagenode_id interval_start interval_seconds action
+
+	field storagenode_id   blob
+	field interval_start   utimestamp
+	field interval_seconds uint32
```

Comment:
Do we need a seperate metric for the interval here? Wouldnt it make more sense to assume it takes until the next interval_start is inserted?

Reply:
For bandwidth not really, if I'm querying let's say for a day, and a storagenode doesn't have any entries vs. it's difficult to understand whether the bandwidth should be accounted for or not.

Reply:
Yeah true, what i am focused on is to keep the indexes as small as possible. But we can change that at any time later

Reply:
cardinality of interval_seconds should be really low so postgres should be able to optimize it really well

Reply:
I trust you on that one 💯